src/ReverbUi.cs

```diff
@@ -12,6 +12,8 @@ namespace ModUI_Reverbed;
     public override void OnLateInitializeMelon()
     {
         // subscribe to shit here I.e. the match start event from the modding ui
+
+        //relevant comment
```

Comment:
i think we should change it to something like this
```suggestion
        //EVEN MORE RELEVANT COMMENT
```